pkg/sql/covering/bench_test.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package covering
+
+import (
+	"encoding/binary"
+	"math/rand"
+	"testing"
+
+	"github.com/cockroachdb/cockroach/pkg/util/timeutil"
+)
+
+func BenchmarkOverlapCoveringMerge(b *testing.B) {
+
+	benchmarks := []struct {
+		// benchmark test
+		name string
+		// rate we would like to "simulate" backups
+		backupRate int
+		// period which backups is going to cover during benchmark
+		backupPeriod int
+		// amount of coverings (ranges) we going to have per each period
+		coveringsPerPeriod int
+	}{
+		{
+			name:               "OneDaysHourlyBackup_10000Coverings",
+			backupRate:         1,
```

Comment:
2¢ I'd simplify this to just two numbers for depth and width, eg. "numLayers" and "elementsPerLayer" instead of adding concepts like rates and periods here, and maybe just put a comment explaining how they relate.

e.g.
```
for numLayers := []int{
1, 
24, // e.g. hourly backups
24*7,  // e.g. hourly backups for a week.
24*4*7, // e.g. backup every 15min for a week
} {
  for elementsPerLayer := range []int{100, 1000, 10000, 100000} {
    doBench(b, numLayers, elementsPerLayer)
  }
}

func doBench(b, numLayers, elementsPerLayer) {
  b.Run("layers=%d,elems=%d", ...)
}
```

Reply:
thanks for feedback, will do the change.